run.sh

```diff
@@ -14,16 +14,17 @@ then
     echo "Great, the simulator is already running!"
 else
     echo "Starting the simulator..."
-    $THIS_DIR/ros/src/styx/unity_simulator_launcher.sh &
+    cd $THIS_DIR/ros/src/styx
+    ./unity_simulator_launcher.sh &
+    cd $THIS_DIR
 fi
 
 # Launch ROS nodes
 docker run --rm=true --tty=true --interactive=true               \
-           --user=$(id --user):$(id --group)                     \
            --volume="/tmp":"/.ros"                               \
            --volume="$THIS_DIR":"$THIS_DIR"                      \
            --workdir="$ROS_DIR"                                  \
-           --network=host                                        \
+            -p 4567:4567 \
```

Comment:
I tried this before and it works.

However without network=host, us people with native Ubuntu cannot use a few extras, for example Rviz, since que cannot communicate with the roscore running inside  Docker.

Is there any problem using network=host?